packages/frontend/app/logging.ts

```diff
@@ -36,9 +37,15 @@ configure({
             layout: { type: 'json', separator: ',' },
         },
     },
-    categories: { default: { appenders: ['fileAppender'], level: 'info' } },
+    categories: {
+        default: { appenders: ['fileAppender'], level: 'info' },
+        development: { appenders: ['console'], level: 'warn' },
```

Comment:
maybe we should also see info level logs in dev mode, might be helpful for debugging?

Reply:
No problem. Will update and merge.